server/src/main/java/org/apache/druid/server/QueryLifecycle.java

```diff
@@ -318,14 +318,12 @@ public void emitLogsAndMetrics(
 
       if (e != null) {
         statsMap.put("exception", e.toString());
-
-        if (e instanceof QueryInterruptedException) {
-          // Mimic behavior from QueryResource, where this code was originally taken from.
-          log.noStackTrace().warn(e, "Exception while processing queryId [%s]", baseQuery.getId());
-          statsMap.put("interrupted", true);
-          statsMap.put("reason", e.toString());
-        }
+        // Mimic behavior from QueryResource, where this code was originally taken from.
```

Comment:
Why this change? I don't think it is correct, since this stuff should only be set if it was a `QueryInterruptedException`

Reply:
Changed to just moving the logging out of the if check

Reply:
The change is so that we can see the exception (and possibly stacktrace) for exceptions other than QueryInterruptedException too